digiroad2-api-oth/src/main/scala/fi/liikennevirasto/digiroad2/ServiceRoadAPI.scala

```diff
@@ -0,0 +1,103 @@
+package fi.liikennevirasto.digiroad2
+
+import fi.liikennevirasto.digiroad2.linearasset.{MaintenanceRoad, PersistedLinearAsset, Properties}
+import org.joda.time.DateTime
+import org.joda.time.format.DateTimeFormat
+import org.json4s.{DefaultFormats, Formats}
+import org.scalatra.ScalatraServlet
+import org.scalatra.json.JacksonJsonSupport
+
+class ServiceRoadAPI(val linearAssetService: LinearAssetOperations, val roadLinkService: RoadLinkService) extends ScalatraServlet with JacksonJsonSupport with AuthenticationSupport {
+
+  override def baseAuth: String = "serviceRoad."
+  override val realm: String = "Service Road API"
+
+  private val Kayttooikeus = "huoltotie_kayttooikeus"
+  private val Huoltovastuu = "huoltotie_huoltovastuu"
+  private val Tiehoitokunta = "huoltotie_tiehoitokunta"
+  private val Nimi = "huoltotie_nimi"
+  private val Osoite = "huoltotie_osoite"
+  private val Postinumero = "huoltotie_postinumero"
+  private val Postitoimipaikka = "huoltotie_postitoimipaikka"
+  private val Puh1 = "huoltotie_puh1"
+  private val Puh2 = "huoltotie_puh2"
+  private val Lisatieto = "huoltotie_lisatieto"
+
+  protected implicit def jsonFormats: Formats = DefaultFormats
+  val typeId = LinearAssetTypes.MaintenanceRoadAssetTypeId
+
+  private val trLinkID = "linkId"
```

Comment:
Prefix should not be tr, because this has nothing to do with TR? Maybe livi instead?